api/geth_backend.go

```diff
@@ -2181,8 +2181,8 @@ func (b *GethStatusBackend) loadNodeConfig(inputNodeCfg *params.NodeConfig) erro
 		}
 	}
 
-	// Start WakuV1 if WakuV2 is not enabled
-	conf.WakuConfig.Enabled = !conf.WakuV2Config.Enabled
+	// TODO: Consider removing the Enabled field from the config as WakuV1 has been removed.
```

Comment:
I guess it still makes some sense, in case you don't want to start waku service at all. E.g. wallet-only case.
Although not used now, makes sense to have in the backend.